utilities/logger/src/index.ts

```diff
@@ -0,0 +1,3 @@
+export {Core} from './core';
+export {Server} from './server';
```

Comment:
There will be no more work around login for this so implementation should be complete.
We may have tickets in case of some issues, but try to cover and write framework style logger. You can check existing logging solutions or how other frameworks doing loggers: 
see feathers.js for example